src/components/TuneLabelInput.vue

```diff
@@ -7,8 +7,13 @@ defineProps<{
 </script>
 
 <template>
-  <div class="tune-label mb-[2px] flex items-center gap-1">
-    <slot />
-    <TuneIconHint :hint="hint" />
+  <div class="tune-label-container flex flex-col">
+    <div class="tune-label mb-[2px] flex items-center gap-1">
+      <slot />
+      <TuneIconHint :hint="hint" />
+    </div>
+    <div v-if="$slots.sublabel" class="tune-sublabel text-sm text-skin-text opacity-60">
```

Comment:
Why are you using a slot instead of a prop?

Reply:
For me `sublabel` looks similar to `label` in their purpose. We use a slot for the label, so a `sublabel` should also be placed in the slot.
But there are no best practices when putting text in a slot or prop, so it's more about taste.

Reply:
Props is more obvious, can you change that?